bug: unable to apply fluentd CRD due to exceeded metadata.annotations length limit

### Describe the issue

As titled, the crd is too long and exceeded metadata.annotations length limit.

The CustomResourceDefinition "fluentds.fluentd.fluent.io" is invalid: metadata.annotations: Too long: must have at most 262144 bytes
make: *** [install] Error 1

```
make install                                                                                                             18:51:19
go get k8s.io/code-generator@v0.30.3
go mod download
/fluent-operator/bin/controller-gen "crd:generateEmbeddedObjectMeta=true" rbac:roleName=manager-role webhook paths="./apis/fluentbit/..." output:crd:artifacts:config=config/crd/bases
/fluent-operator/bin/controller-gen "crd:generateEmbeddedObjectMeta=true" rbac:roleName=manager-role webhook paths="./apis/fluentd/..." output:crd:artifacts:config=config/crd/bases
/fluent-operator/bin/controller-gen "crd:generateEmbeddedObjectMeta=true" rbac:roleName=manager-role webhook paths="./apis/fluentbit/..." output:crd:artifacts:config=charts/fluent-operator/charts/fluent-bit-crds/crds
/fluent-operator/bin/controller-gen "crd:generateEmbeddedObjectMeta=true" rbac:roleName=manager-role webhook paths="./apis/fluentd/..." output:crd:artifacts:config=charts/fluent-operator/charts/fluentd-crds/crds
kubectl kustomize config/crd/bases/ | sed -e '/creationTimestamp/d' > manifests/setup/fluent-operator-crd.yaml
kubectl kustomize manifests/setup/ | sed -e '/creationTimestamp/d' > manifests/setup/setup.yaml
/fluent-operator/bin/kustomize build config/crd/bases/ | kubectl apply -f -
customresourcedefinition.apiextensions.k8s.io/clusterfilters.fluentbit.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/clusterfilters.fluentd.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/clusterfluentbitconfigs.fluentbit.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/clusterfluentdconfigs.fluentd.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/clusterinputs.fluentbit.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/clusterinputs.fluentd.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/clustermultilineparsers.fluentbit.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/clusteroutputs.fluentbit.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/clusteroutputs.fluentd.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/clusterparsers.fluentbit.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/collectors.fluentbit.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/filters.fluentbit.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/filters.fluentd.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/fluentbitconfigs.fluentbit.fluent.io unchanged
Warning: Detected changes to resource fluentbits.fluentbit.fluent.io which is currently being deleted.
customresourcedefinition.apiextensions.k8s.io/fluentbits.fluentbit.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/fluentdconfigs.fluentd.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/inputs.fluentd.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/multilineparsers.fluentbit.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/outputs.fluentbit.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/outputs.fluentd.fluent.io unchanged
customresourcedefinition.apiextensions.k8s.io/parsers.fluentbit.fluent.io unchanged
The CustomResourceDefinition "fluentds.fluentd.fluent.io" is invalid: metadata.annotations: Too long: must have at most 262144 bytes
make: *** [install] Error 1

```

### To Reproduce

run `make install` in any environment

### Expected behavior

the fluentd CRD should be applied successfully

### Your Environment

```markdown
- Fluent Operator version: latest master
- Container Runtime: N/A
- Operating system: N/A
- Kernel version: N/A
```


### How did you install fluent operator?

_No response_

### Additional context

_No response_

--- a/Makefile
+++ b/Makefile
@@ -142,13 +142,13 @@ push-amd64:
 ##@ Deployment
 
 install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd/bases/ | kubectl apply -f -
+	$(KUSTOMIZE) build config/crd/bases/ | kubectl create -f -
 
 uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/crd/bases/ | kubectl delete -f -
 
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
-	kubectl apply -f manifests/setup/setup.yaml
+	kubectl create -f manifests/setup/setup.yaml
 
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config.
 	kubectl delete -f manifests/setup/setup.yaml

